### ISSUE TITLE
macOS: delete old nightly builds in application support folder

### ISSUE THREAD

**Issue by EricE549X** [StefanLobbenmeier/yt-dlp-compose#85]
last night, i was clearing out application support files to save space (i have a 256GB drive) [don't worry, the files regenerate if the app is downloaded again], and i found over three months/a gigabyte's worth of nightly yt-dlp builds in the yt-dlp-compose folder. once a new build is installed, the old one(s) should be deleted.

**steps to locate these builds:**
1. open finder and press shift-command-G, or click "go" → "go to folder" in the menu bar
2. alternatively, press and hold (but don't force click) or right click the finder icon in the dock and click "go to folder"
3. type in "/Users/[username]/Library/Application Support" (the user library is separate from the drive library and isn't accessible through the finder window)
4. scroll down to "yt-dlp-compose" folder
5. go to "binaries" → "yt-dlp" → "yt-dlp-nightly-builds"
6. see builds

**Comment by StefanLobbenmeier** [comment]
Good point - I definitely plan on doing that. As a workaround for now you could use the latest version and switch to the stable branch, which gets an update around once a month.

Background for keeping the old versions is that o plan on being able to revert to previous versions, but that’s obviously not complete yet 😄 I should have more time again in December

**Comment by EricE549X** [comment]
yes
